google_compute_engine/networking/ip_forwarding/ip_forwarding_utils.py

```diff
@@ -15,11 +15,7 @@
 
 """Utilities for configuring IP address forwarding."""
 
-import re
-import subprocess
-
-IP_REGEX = re.compile(r'\A(\d{1,3}\.){3}\d{1,3}\Z')
-IP_ALIAS_REGEX = re.compile(r'\A(\d{1,3}\.){3}\d{1,3}/\d{1,2}\Z')
+from google_compute_engine.compat import distro_utils
 
 
 class IpForwardingUtils(object):
```

Comment:
I think you can replace this class with the following

```
class IpForwardingUtils(distro_utils.IpForwardingUtils):
  """Deprecated. Overridden for backwards compatibility."""
  pass
```

Reply:
To do this I had to add in each distro specific file the following:

    +from google_compute_engine.distro_lib.ip_forwarding_utils import \
    +    IpForwardingUtilsIproute as IpForwardingUtils

And I removed the function `def IpForwardingUtils` from `class Utils`, please let me know if this is ok or if you had another idea in mind.